packages/opentelemetry-exporter-collector-grpc/src/util.ts

```diff
@@ -108,8 +108,11 @@ export function send<ExportItem, ServiceRequest>(
 }
 
 export function validateAndNormalizeUrl(url: string): string {
+  if (!url.match(/^([\w]{1,8}):\/\//)) {
```

Comment:
```suggestion
  const hasProtocol = url.match(/^([\w]{1,8}):\/\//;
  if (!hasProtocol) {
```

nit: self documenting var name since regex is hard to read